### PR TITLE
Add prerequisites

### PR DESCRIPTION
**Pull request by proteusvacuum**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,10 +8,19 @@ For deploying this tool to a production server, see [commcare-sync-ansible](http
...
 ## Developer Setup
 
+### Prerequisites
+- Python 3.8
+    On Ubuntu:
+    ```bash
+    sudo apt-get install python3.8 python3.8-dev
+    ```
+- Postgres
+- Redis
```
Comment (proteusvacuum) [inline README.md:18]: would we think of adding these as docker packages here?

Comment (czue) [reply README.md:18]: What do you mean, exactly? In my local env I just use the same docker images as HQ.

File: README.md [hunk 2]
```diff
@@ -8,10 +8,19 @@ For deploying this tool to a production server, see [commcare-sync-ansible](http
 
 ## Developer Setup
 
+### Prerequisites
+- Python 3.8
+    On Ubuntu:
+    ```bash
+    sudo apt-get install python3.8 python3.8-dev
+    ```
+- Postgres
```
Comment (czue) [inline README.md:17]: I *think* sqlite / any DB works fine, and unlike HQ this project is more likely to be hosted on a different stack (though thus far we have always included our own postgres for convenience). Any reason you feel Postgres is a requirement?

**Review by czue — APPROVED** [review]
thanks! no blockers, just some questions

File: README.md (hunk 1 above)
Comment (proteusvacuum) [reply README.md:18]: so did I, but I feel like this is a completely standalone product, so we should probably have instructions on how to run this without HQ locally.

Comment (czue) [reply README.md:18]: That makes sense. I would have thought the simplest thing to do is just install postgres natively, though I have no objections to adding docker instructions as well.

**Review by czue — APPROVED** [review]

**Pull request merged**